homeassistant/components/trafikverket_weatherstation/config_flow.py

```diff
@@ -120,3 +125,57 @@ async def async_step_reauth_confirm(
             data_schema=vol.Schema({vol.Required(CONF_API_KEY): cv.string}),
             errors=errors,
         )
+
+    async def async_step_reconfigure(
+        self, entry_data: Mapping[str, Any]
+    ) -> ConfigFlowResult:
+        """Handle re-configuration with Trafikverket."""
+
+        self.entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
```

Comment:
I think we have a new helper for this :)

Reply:
Yes, second time @epenet puts in some helper.
I think he tries to delay this PR on purpose 🤣

Reply:
@epenet, he found out, abort mission

Reply:
Take a look at #127316...
I suggest to merge that one and maybe adjust this one afterwards.

Reply:
No, I'll fix this one later tonight. That's fine 👍 
Just joking a bit...